codesign.py

```diff
@@ -757,6 +795,10 @@ def main(args, bucket_prefix):
 CODESIGN_USERNAME = os.environ['CODESIGN_USERNAME']
```

Comment:
can we delete this?

Reply:
umm This seem to be used in a lot of places. Maybe I am missing something, why would we like to delete it?